dts/bindings/pinctrl/nxp,imx8m-pinctrl.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The node has the 'pinctrl' node label set in MCUX SoC's devicetree. These
+  nodes can be autogenerated using the MCUXpresso config tools combined with
+  the imx_dts_gen.py script in NXP's HAL. The mux, mode, input, daisy, and cfg
+  fields in a group select the pins to be configured, and the remaining
+  devicetree properties set configuration values for those pins
+  for example, here is an group configuring UART2 pins:
+
+  group0 {
+    pinmux = <&iomuxc_uart2_rxd_uart_rx_uart2_rx,
+      &iomuxc_uart2_txd_uart_tx_uart2_tx>;
+    drive-strength = "40-ohm";
+    slew-rate = "slow";
+  };
```

Comment:
nit: fix alignment here

Reply:
I'm assuming you're referring to the indentation here (that additional indentation on line 14 was intentional)- I've increased the indentation here to improve readability